website/pages/en/network/indexing.mdx

```diff
@@ -569,23 +569,23 @@ For example, if the global rule has a `minStake` of **5** (GRT), any subgraph de
 
 Data model:
 
-```graphql
-type IndexingRule {
-    identifier: string
-    identifierType: IdentifierType
-    decisionBasis: IndexingDecisionBasis!
-    allocationAmount: number | null
-    allocationLifetime: number | null
-    autoRenewal: boolean
-    parallelAllocations: number | null
-    maxAllocationPercentage: number | null
-    minSignal: string | null
-    maxSignal: string | null
-    minStake: string | null
-    minAverageQueryFees: string | null
-    custom: string | null
-    requireSupported: boolean | null
-  }
+```text
+Type IndexingRule {
```

Comment:
I am not sure which language it's so I marked type with `Type` as in codeblock on line 632

Reply:
this is assemblyscript?